client/src/components/QandA/QuestionList.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
```

Comment:
I did - this is just my temp fix since I haven't gone through implementing prop-types yet in any of my files as mentioned. I will be sure to do so in a later update!